src/test/java/net/bhardy/bizzo/billing/PolicyBuilderTest.java

```diff
@@ -33,233 +33,197 @@
     }
 
     @Test
     public void isDueOn_weeklyFiltered() {
         BillingPolicy policy = BillingPolicy.builder()
-                .sameDayEveryWeek(FRIDAY)
+                .weeklyOnDay(FRIDAY)
                 .filter(not(daysOfWeek(SATURDAY, SUNDAY)))
                 .action(NEXT_DAY)
                 .build();
 
-        LocalDate oneSaturday = LocalDate.of(2017, 8, 5);
-        assertFalse(policy.isDueOn(oneSaturday));
-        LocalDate monday = LocalDate.of(2017, 8, 7);
-        assertFalse(policy.isDueOn(monday));
+        assertFalse(policy.isDueOn(saturday5th));
+        assertFalse(policy.isDueOn(monday7th));
     }
 
     @Test
     public void isDueOn_monthlySimple() {
         BillingPolicy policy = BillingPolicy.builder()
-                .sameDayEveryMonth(5)
+                .monthlyOnDay(5)
                 .build();
 
-        LocalDate oneSaturday = LocalDate.of(2017, 8, 5);
-        assertTrue(policy.isDueOn(oneSaturday));
+        assertTrue(policy.isDueOn(saturday5th));
+        assertTrue(policy.isDueOn(LocalDate.of(2017, 12, 5)));
     }
 
     @Test
     public void isDueOn_monthlyFiltered() {
         BillingPolicy policy = BillingPolicy.builder()
-                .sameDayEveryMonth(5)
+                .monthlyOnDay(5)
                 .filter(not(daysOfWeek(SATURDAY, SUNDAY)))
                 .action(NEXT_DAY)
                 .build();
 
-        LocalDate oneSaturday = LocalDate.of(2017, 8, 5);
-        assertFalse(policy.isDueOn(oneSaturday));
-        LocalDate maybeSunday = LocalDate.of(2017, 8, 6);
-        assertFalse(policy.isDueOn(maybeSunday));
-        LocalDate shouldBeMonday = LocalDate.of(2017, 8, 7);
-        assertTrue(policy.isDueOn(shouldBeMonday));
+        assertFalse(policy.isDueOn(saturday5th));
+        assertFalse(policy.isDueOn(sunday6th));
+        assertTrue(policy.isDueOn(monday7th));
     }
 
     @Test
     public void upcomingDueDates_dailySimple() {
         BillingPolicy policy = BillingPolicy.builder()
-                .everyDay()
+                .daily()
                 .build();
 
-        LocalDate oneSaturday = LocalDate.of(2017, 8, 5);
-        List<LocalDate> nextBills = policy.upcomingDueDates(oneSaturday, 2);
-
-        LocalDate expected1 = LocalDate.of(2017, 8, 5);
-        LocalDate actual1 = nextBills.get(0);
-        assertEquals(expected1, actual1);
-        assertEquals(SATURDAY, actual1.getDayOfWeek());
+        List<LocalDate> nextBills = policy.upcomingDueDates(saturday5th, 2);
 
-        LocalDate expected2 = LocalDate.of(2017, 8, 6);
-        LocalDate actual2 = nextBills.get(1);
-        assertEquals(expected2, actual2);
-        assertEquals(SUNDAY, actual2.getDayOfWeek());
+        assertEquals(saturday5th, nextBills.get(0));
+        assertEquals(sunday6th, nextBills.get(1));
     }
 
     @Ignore  // this reveals an underlying bug we need to fix
     @Test
     public void upcomingDueDates_dailyFiltered() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)